frontend/src/app/containers/ExperimentPage.js

```diff
@@ -146,12 +147,16 @@ export class ExperimentDetail extends React.Component {
     return (
       <section className="incompatible-addons">
         <header>
-          <h3 data-l10n-id="incompatibleHeader">
-            This experiment may not be compatible with add-ons you have installed.
-          </h3>
-          <p data-l10n-id="incompatibleSubheader">
-            We recommend <a href={helpUrl}>disabling these add-ons</a> before activating this experiment:
-          </p>
+          <Localized id="incompatibleHeader">
+            <h3>
+              This experiment may not be compatible with add-ons you have installed.
+            </h3>
+          </Localized>
+          <Localized id="incompatibleSubheader">
+            <p>
+              We recommend <a href={helpUrl}>disabling these add-ons</a> before activating this experiment:
```

Comment:
Great, thanks a lot. However what if the contents of the anchor needs to be localized too?